config/docker/catalog-standalone-defs/apiDocs/service1-instance1_org.zowe v1.0.0_default.json

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
Could we somehow merge the local and docker versions of these jsons? If you think it is a good idea ...

Reply:
I thought about it, my opinion is that it could change later, more tests, etc and having separate versions makes sense for that situation, but it's not a strong opinion.
On the other hand, these are in different directories depending on where they are used, so it could lead to confusion if we call the `local` version from the docker-based tests